lib/game.rb

```diff
@@ -0,0 +1,21 @@
+class Game
+
+  def initialize
+    @rolls = []
+  end
+
+  def roll(pins)
+    @rolls << pins
+  end
+
+  def score
+    result = 0
+    roll_index = 0
+    20.times do
```

Comment:
May be worth setting this to `10.times{}` , as there are 10 frames - this currently assumes 2 balls on every frame regardless of outcome.

Will help for identifying games where strikes are made and less overall balls are thrown.